data/apps.json

```
@@ -195,15 +195,25 @@
       "user": "LMMS",
       "repo": "lmms"
     }
+  },
+    {
+    "name": "Musescore 4",
```

I'd prefer to keep the name only (without being attached to any specific version)

```suggestion
    "name": "Musescore",
```